index.html

```diff
@@ -2845,7 +2834,7 @@ <h3>Trust Model</h3>
 it received. To establish this trust, a [=credential=] is expected to either:
             <ul>
               <li>
-Include a <a href="#proofs-signatures">proof</a> establishing that the
+Secure the [=credential=] with a <a href="#proofs-signatures">proof</a> establishing that the
 [=issuer=] generated the [=credential=] (that is, it is a
 [=verifiable credential=]), or
```

Comment:
We will want to just reword this to refer to the "Securing Mechanisms" section. This is not blocking feedback, and it's editorial, but while we're in here we might as well change this/update it to point to a better place.